kinto_emailer/tests/test_includeme.py

```diff
@@ -80,12 +87,9 @@ def test_get_collection_record(self):
 
     def test_send_notification_is_called_on_new_record(self):
         with mock.patch('kinto_emailer.send_notification') as mocked:
-            app = self._get_test_app({
-                'includes': ['kinto.plugins.default_bucket', 'kinto_emailer'],
-                'emailer.sender': 'kinto.email@restmail.net',
-            })
-            app.post_json('/buckets/default/collections/foobar/records',
-                          headers={'Authorization': 'Basic bmF0aW06'})
+            import pdb; pdb.set_trace()
+            self.app.post_json('/buckets/default/collections/foobar/records',
```

Comment:
I don't understand why this call to the default_bucket is forbidden as if the default bucket was not activated, but it actually is :(